openBCIBoard.js

```diff
@@ -66,6 +67,12 @@ function OpenBCIFactory () {
   *              `v1` - Firmware Version 1 (Default)
   *              `v2` - Firmware Version 2
   *
+  *     - `simulatorFragmentation` {String} - Specifies how to break packets to simulate fragmentation.
+  *          3 Possible Options:
+  *              `Random` - output randomly-lengthed chunks of data (Default)
```

Comment:
whoops i was going off `boardType`. Want to change line noise in that case too haha. For sure list as a breaking change in the `changelog.md`. This aligns with the semi standard adoption of not starting stuff with upper case too.